### PR TITLE
revert all 1.0.0 changes

### PR DESCRIPTION
**Pull request by samjsong**
###### Description

Reverts all `1.0.0` changes for `0.16.0`
https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/426/
https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/446/
https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/451/
https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/453/
https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/429/
https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/428/

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

**Comment by sumo-drosiek** [comment]
I suppose we have to update otelcol image after fixes?
@perk-sumo @pmm-sumo

**Comment by pmm-sumo** [comment]
> I suppose we have to update otelcol image after fixes?
> @perk-sumo @pmm-sumo

That is right, I would also like to make some small changes to the default config. Something like:

```
    image:
      name: "sumologic/opentelemetry-collector"
      tag: "0.2.6.5"
      pullPolicy: Always
  config:
    receivers:
      opencensus:
        endpoint: "0.0.0.0:55678"
      jaeger:
        protocols:
          grpc:
            endpoint: "0.0.0.0:14250"
          thrift_http:
            endpoint: "0.0.0.0:14268"
      zipkin:
        endpoint: "0.0.0.0:9411"
```

**Comment by sumo-drosiek** [comment]
I think that changes should be independent of this PR, in case of reverting it on github

File: deploy/docs/existingPrometheusDoc.md [hunk 1]
```diff
@@ -19,15 +19,7 @@ curl -LJO https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-colle
...
 ```bash
-helm install sumologic/sumologic --name collection --namespace sumologic -f values.yaml --set sumologic.accessId=<SUMO_ACCESS_ID> --set sumologic.accessKey=<SUMO_ACCESS_KEY> --set sumologic.clusterName=<MY_CLUSTER_NAME> 
-```
-
-If the prometheus-operator is installed in a different namespace than where the Sumo Logic Solution is deployed you need to copy one of the configmaps that exposes the release name, which is used in the remote write URLs.
-
-For example: 
-If the Sumo Logic Solution is deployed in `<source-namespace>` and the existing prometheus-operator is in `<destination-namespace>`, run the below command: 
-```bash
-kubectl get configmap sumologic-configmap — namespace=<source-namespace> — export -o yaml |\ kubectl apply — namespace=<destination-namespace> -f -
```
Comment (vsinghal13) [inline deploy/docs/existingPrometheusDoc.md]: This is still required as of v0.15.

File: deploy/docs/standAlonePrometheus.md [hunk 2]
```diff
@@ -4,17 +4,5 @@ Update your Prometheus configuration file’s `remote_write` section, as per the
 
 * `writeRelabelConfigs:` change to `write_relabel_configs:`
 * `sourceLabels:` change to `source_labels:`
-*  Modify remote URLs in the `remoteWrite` section of the `prometheus-overrides.yaml` file
```
Comment (vsinghal13) [inline deploy/docs/standAlonePrometheus.md]: Same steps would be required for v0.16

**Pull request merged**